src/plugins/dashboard/public/dashboard_app/top_nav/use_dashboard_menu_items.tsx

```diff
@@ -12,6 +12,7 @@ import { Dispatch, SetStateAction, useCallback, useMemo, useState } from 'react'
 import { ViewMode } from '@kbn/embeddable-plugin/public';
 import { TopNavMenuData } from '@kbn/navigation-plugin/public';
 
+import { HttpStart } from '@kbn/core/public';
```

Comment:
Big 'ol nit: the gap is on the wrong side of this dependency import:
```suggestion
import { HttpStart } from '@kbn/core/public';

```